ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -0,0 +1,245 @@
+import Component
+import ComposableArchitecture
+import Model
+import Styleguide
+import SwiftUI
+
+public struct TimetableDetailScreen: View {
+    enum Const {
+        static let imageSize: CGFloat = 40
+        static let iconSize: CGFloat = 20
+        static let footerIconSize: CGFloat = 24
+    }
+
+    let store: Store<ViewState, ViewAction>
+
+    struct ViewState: Equatable {
+        var timetable: TimetableItem
+    }
+
+    enum ViewAction {}
+
+    public var body: some View {
+        ZStack {
+            ScrollView {
+                WithViewStore(store) { viewStore in
+                    Text(viewStore.timetable.title.jaTitle)
+                        .font(.title)
+                        .foregroundColor(AssetColor.Base.primary.color)
+                        .lineLimit(nil)
+                        .padding(.bottom, 12)
+
+                    speakers(viewStore.timetable.speakers)
+
+                    descriptions(timetable: viewStore.timetable)
+
+                    body(timetable: viewStore.timetable)
+                }
+                .padding(.horizontal, 16)
+            }
+
+            fixedFooter(
+                tapSlide: {
+                    // TODO: FIXME
```

Comment:
fix when I made reducer for detail

Reply:
Is this todo will be done in another PR?

Reply:
yup